.circleci/README.md

```diff
@@ -0,0 +1,100 @@
+## CircleCI
+
+#### config.yml overview:
```

Comment:
The rest of the README looks useful, but this section is almost guaranteed to go out of sync, and doesn't provide too much value over just looking at the config directly. I would suggest removing it.

Reply:
Agree with removal - would just add these in the config.yml itself as appropriate